README.md

```diff
@@ -131,6 +122,36 @@ Notes:
   However, the `source_code_url` already provides an (unauthenticated) 
   way to point to the author(s). 
 
+## Developing / Building from source
+
+### Requirements
+
+You need to have installed:
+- `rust` (https://www.rust-lang.org/tools/install)
+- `node` (https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- `pnpm` (https://pnpm.io/installation)
+
+### Building the frontend
+
+The bot uses a WebXDC app as "frontend" to interact with users.
+Go to the `frontend` folder and install the packages using `pnpm install`.
```

Comment:
did you drop the extra information about how to get "pnpm"? 
for me only "npm install -g pnpm" worked, not the "npx" thingie btw.

Reply:
I pointed to the "installation instructions" of pnpm in the requirements section